cypress/e2e/certificate-provider/enter-date-of-birth.cy.js

```diff
@@ -1,78 +1,97 @@
 describe('Enter date of birth', () => {
-    beforeEach(() => {
-        cy.visit('/fixtures/certificate-provider?redirect=/enter-date-of-birth');
-    });
+    describe('can be completed by a', () => {
+        it('lay certificate provider', () => {
+            cy.visit('/fixtures/certificate-provider?redirect=/enter-date-of-birth');
+            cy.checkA11yApp();
 
-    it('can be completed', () => {
-        cy.checkA11yApp();
+            cy.get('#f-date-of-birth').type('1');
+            cy.get('#f-date-of-birth-month').type('2');
+            cy.get('#f-date-of-birth-year').type('1990');
 
-        cy.get('#f-date-of-birth').type('1');
-        cy.get('#f-date-of-birth-month').type('2');
-        cy.get('#f-date-of-birth-year').type('1990');
+            cy.contains('button', 'Continue').click();
 
-        cy.contains('button', 'Continue').click();
+            cy.url().should('contain', '/confirm-your-details');
+        });
 
-        cy.url().should('contain', '/confirm-your-details');
-    });
 
-    it('can be over 100', () => {
-        cy.checkA11yApp();
+        it('professional certificate provider', () => {
+            cy.visit('/fixtures/certificate-provider?redirect=/enter-date-of-birth&as-prof-cp=1');
```

Comment:
could you make this `professional=1` or `relationship=professional`, I kind of want to avoid building up a bunch of inconsistently named params like we had before

Reply:
Yep, `relationship=professional` works for me.